conf/machine/include/amlogic-a311d.inc

```diff
@@ -5,3 +5,5 @@ require conf/machine/include/amlogic-meson64.inc
 MACHINE_FEATURES = "serial usbhost sdio screen wifi"
 
 SOC_FAMILY_append = ":meson-g12b"
+
+KERNEL_CLASSES_khadas-vim3 ?= "kernel-uimage-meson"
```

Comment:
You should move this to amlogic-meson64.inc and change it to KERNEL_CLASSES_meson-gx ?= "kernel-uimage-meson"

Reply:
I'll look into this...

Reply:
Hey @superna9999 
All review comments (apart from the single .scc file) resolved in the latest pull request.